bpfd/src/server/pull_bytecode.rs

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: (MIT OR Apache-2.0)
+// Copyright Authors of bpfd
+use flate2::read::GzDecoder;
+use log::debug;
+use oci_distribution::{client, manifest, secrets::RegistryAuth, Client, Reference};
+use serde::Deserialize;
+use serde_json::Value;
+use tar::Archive;
+
+const CONTAINERIZED_BYTECODE_PATH: &str = "/etc/bpfd/bytecode/";
```

Comment:
This should be in `/var`

Reply:
So this might break @Billy99's work... If we're not running with `sudo` will we have the ability to create/ put files in `/var/bpfd/bytecode`?

Reply:
I had  code changes pending on my branch to change the user from `bpfctl` to `bpfd-clients`. I can change the directory with those changes and make sure it has the correct owner.

Reply:
Ack thanks!

Reply:
See #118
FYI: I change the name if needed.